components/ipfs/ipfs_node_traffic_recognizer.cc

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ipfs/ipfs_node_traffic_recognizer.h"
+
+#include "brave/components/ipfs/buildflags/buildflags.h"
+#include "components/version_info/channel.h"
+
+#if BUILDFLAG(ENABLE_IPFS_LOCAL_NODE)
+#include "brave/components/ipfs/ipfs_constants.h"
+#include "brave/components/ipfs/ipfs_ports.h"
+#include "url/origin.h"
+#endif
+
+namespace {
+const int kDefaultKuboAPIPort = 5001;
+}  // namespace
+
+namespace ipfs {
+
+bool IpfsNodeTrafficRecognizer::IsKuboRelatedPort(
+    const GURL& request_url,
+    const version_info::Channel& channel) {
```

Comment:
I think we should check every port for every version. Since extension may try accessing kubo beta or kubo nightly

Reply:
Also i think we should protect swarm and gateway ports for every channel

Reply:
done